packages/back-end/src/util/secrets.ts

```diff
@@ -22,11 +22,27 @@ export const UPLOAD_METHOD = (() => {
   return "local";
 })();
 
-export let MONGODB_URI =
-  process.env.MONGODB_URI ??
-  (prod ? "" : "mongodb://root:password@localhost:27017/test?authSource=admin");
```

Comment:
Your code is missing this logic.  If it is still not defined after the alternate mongo db env vars AND not prod, then we want to use this as the default.

Reply:
That logic is moved down lower on line 50. Did I get that wrong?